### ISSUE TITLE
TimeStampedModel breaks django migrations in version 1.5.6

### ISSUE THREAD

**Issue by zeraien** [django-extensions/django-extensions#735]
I upgraded from 1.5.5 to 1.5.6 and now whenever I try to run `makemigrations` the following error occurs for models that inherit `TimeStampedModel`:

```pytb
ValueError: Cannot serialize: <class django.db.models.fields.NOT_PROVIDED at 0x1076a7ae0>
There are some values Django cannot serialize into migration files.
For more, see https://docs.djangoproject.com/en/dev/topics/migrations/#migration-serializing
```

It seems to be related to the "default" value of the `DateTimeField` and occurs during in the Django migration system.

The change that causes the problem is here:
https://github.com/django-extensions/django-extensions/compare/1.5.5...1.5.6#diff-8e3fa7c3b02c3419c7374340925fe214L202

Tested to affect Django 1.7.4 up to 1.8.4

Full stack trace
```pytb
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Volumes/Krieger/Users/zeraien/Development/ingager/results/env-ingager_results/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/Volumes/Krieger/Users/zeraien/Development/ingager/results/env-ingager_results/lib/python2.7/site-packages/django/core/management/__init__.py", line 377, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Volumes/Krieger/Users/zeraien/Development/ingager/results/env-ingager_results/lib/python2.7/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/Volumes/Krieger/Users/zeraien/Development/ingager/results/env-ingager_results/lib/python2.7/site-packages/django/core/management/base.py", line 338, in execute
    output = self.handle(*args, **options)
  File "/Volumes/Krieger/Users/zeraien/Development/ingager/results/env-ingager_results/lib/python2.7/site-packages/django/core/management/commands/makemigrations.py", line 124, in handle
    self.write_migration_files(changes)
  File "/Volumes/Krieger/Users/zeraien/Development/ingager/results/env-ingager_results/lib/python2.7/site-packages/django/core/management/commands/makemigrations.py", line 152, in write_migration_files
    migration_string = writer.as_string()
  File "/Volumes/Krieger/Users/zeraien/Development/ingager/results/env-ingager_results/lib/python2.7/site-packages/django/db/migrations/writer.py", line 132, in as_string
    operation_string, operation_imports = OperationWriter(operation).serialize()
  File "/Volumes/Krieger/Users/zeraien/Development/ingager/results/env-ingager_results/lib/python2.7/site-packages/django/db/migrations/writer.py", line 88, in serialize
    arg_string, arg_imports = MigrationWriter.serialize(arg_value)
  File "/Volumes/Krieger/Users/zeraien/Development/ingager/results/env-ingager_results/lib/python2.7/site-packages/django/db/migrations/writer.py", line 334, in serialize
    return cls.serialize_deconstructed(path, args, kwargs)
  File "/Volumes/Krieger/Users/zeraien/Development/ingager/results/env-ingager_results/lib/python2.7/site-packages/django/db/migrations/writer.py", line 240, in serialize_deconstructed
    arg_string, arg_imports = cls.serialize(arg)
  File "/Volumes/Krieger/Users/zeraien/Development/ingager/results/env-ingager_results/lib/python2.7/site-packages/django/db/migrations/writer.py", line 410, in serialize
    "topics/migrations/#migration-serializing" % value
ValueError: Cannot serialize: <class django.db.models.fields.NOT_PROVIDED at 0x1076a7ae0>
There are some values Django cannot serialize into migration files.
For more, see https://docs.djangoproject.com/en/dev/topics/migrations/#migration-serializing
```

**Comment by aeby** [comment]
Possible duplicate of #733

**Comment by zeraien** [comment]
Yes it certainly seems so.

**Comment by trbs** [comment]
closed as duplicate of #733
